### PR TITLE
Add baidu in analytics provider, add tip for enabling analytics plugin

### PR DESCRIPTION
**Pull request by themez**
analytics only works when `safe` is true, it's better to put a tip there.

### CODE REVIEW

**Comment by groundh0g** [comment]
Related issue #84. site.safe may be obsoleted.

File: _config.yml [hunk 1]
```diff
@@ -95,6 +95,7 @@ JB :
       colorscheme: light
    
   # Settings for analytics helper
+  # Set 'site.safe' to true to enable analytics plugin
```
Comment (groundh0g) [inline _config.yml]: @themez we've moved away from using site.safe to decide if the site is in production mode. I'm killing this line and merging the rest.

Comment (groundh0g) [inline _config.yml]: @themez, crap. I can't edit inline. I will have to resubmit this using my account, or you can make the tweak on your end and I'll merge. Lemme know.

Comment (groundh0g) [inline _config.yml]: @themez, double crap. This now has merge conflicts. Will have to create new PR. If you don't have time or desire, I'll handle it. Thanks for contribution!

**Comment by groundh0g** [comment]
Isolated. New analytics provider. Low risk. BUT ... Cannot merge now. See line-item comments.

File: _config.yml (hunk 1 above)
Comment (themez) [inline _config.yml]: Hi ground0g, I've already deleted my fork repo, cannot make a new pr now.. It would be good if you can fix it with your account.

Comment (groundh0g) [inline _config.yml]: Will do. Thanks for the contribution! Sorry it's taken so long to review.

**Comment by groundh0g** [comment]
From @themez:

> Hi ground0g, I've already deleted my fork repo, cannot make a new pr now.. It would be good if you can fix it with your account.

Will do. Thanks for the contribution! Sorry it's taken so long to review.

**Comment by groundh0g** [comment]
Made edits by hand in local repo. Pushing this weekend.

**Pull request closed without merging**